src/EnergyPlus/DXCoils.cc

```diff
@@ -9504,7 +9502,7 @@ Label50: ;
 						OutletAirDryBulbTemp = OutletAirDryBulbTempSat;
 						OutletAirHumRat = PsyWFnTdbH( OutletAirDryBulbTemp, OutletAirEnthalpy, RoutineNameHighSpeedOutlet );
 					}
-					//LSOutletAirRH = PsyRhFnTdbWPb(OutletAirDryBulbTemp,OutletAirHumRat,OutdoorPressure,'CalcMultiSpeedDXCoil:highspeedoutlet')
```

Comment:
The whole issue here is that OutletAirRH is never used for anything. Yes, if RH>100 the Psyc function corrects it to 100%, warns, and moves on. But that value is never used again. The other properties, H, T, w, are recalculated so calling the RH psyc routine is redundant.